### PR TITLE
Add Bug Report Template

### PR DESCRIPTION
**Pull request by EvanR-Dev**
This is what users will be redirected to

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    id: error-logs
+    attributes:
+      label: Error Logs
+      description: If applicable, include error logs or messages related to the bug.
+
+  - type: input
+    id: suggested-fix
+    attributes:
+      label: Suggested Fix
+      description: If you have suggestions on how to fix the bug, please provide them here.
```
Comment (vrdmr) [inline .github/ISSUE_TEMPLATE/bugreport.yml:71]: Newline at the end

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    attributes:
+      label: Function App Name
+      description: Please give the name of the function app being used.
+
+  - type: input
+    id: tunnel-id
+    attributes:
+      label: Tunnel ID
+      description: If applicable, please provide the Tunnel ID. It is located in the error message or at the edge of the status bar, typically in the form [word]-[word]-[id].[region]. Example - joyful-horse-y7jqra5.use
+
```
Comment (vrdmr) [inline .github/ISSUE_TEMPLATE/bugreport.yml:42]: Do you think asking region would help? or tunnel id should be good for now.

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
+name: Azure Functions VSCode Web Bug Report
```
Comment (vrdmr) [inline .github/ISSUE_TEMPLATE/bugreport.yml:1]: Add another "feature request" template as well.

**Review by vrdmr — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bugreport.yml (hunk 2 above)
Comment (EvanR-Dev) [reply .github/ISSUE_TEMPLATE/bugreport.yml:42]: The ID will include the region suffix as a part of it. Also the function app name will give this away too

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        4. Observe [describe what you see, e.g., an error message or unexpected behavior].
+        5. Include any additional steps or details that may be relevant.
+
+  - type: input
+    id: function-app-name
+    attributes:
+      label: Function App Name
+      description: Please give the name of the function app being used.
+
+  - type: input
```
[the remark this answers is not in the document]
Comment (vrdmr) [reply .github/ISSUE_TEMPLATE/bugreport.yml:37]: For App name I would not recommend getting it

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        3. Scroll down to [specific location].
+        4. Observe [describe what you see, e.g., an error message or unexpected behavior].
+        5. Include any additional steps or details that may be relevant.
+
+  - type: input
+    id: function-app-name
+    attributes:
+      label: Function App Name
+      description: Please give the name of the function app being used.
+
```
Comment (vrdmr) [inline .github/ISSUE_TEMPLATE/bugreport.yml:36]:
```suggestion
  - type: textarea
     id: function-app-name
     attributes:
       label: Function app name
       placeholder: https://github.com/Azure/azure-functions-host/wiki/Sharing-Your-Function-App-name-privately
       description: Optionally share your function app name.
```

**Pull request merged**